modules/plugins/statusline/lualine/config.nix

```diff
@@ -20,13 +20,13 @@ in {
         extensions = ["nvim-tree"];
       };
     })
-    (mkIf (breadcrumbsCfg.enable && breadcrumbsCfg.source == "nvim-navic") {
+    (mkIf (bcfg.enable && bcfg.source == "nvim-navic" && bcfg.lualine.winbar.enable) {
```

Comment:
Maybe group the 2  enable options together, makes a little more sense, eg.: `bcfg.enable && bcfg.lualine.winbar.enable && bcfg.source == "nvim-navic`